pixeltable/catalog/catalog.py

```diff
@@ -168,10 +168,11 @@ class Catalog:
 
     _instance: Catalog | None = None
 
-    # cached TableVersion instances; key: [id, version]
+    # cached TableVersion instances; key: [id, version, anchor_tbl_id]
     # - mutable version of a table: version == None (even though TableVersion.version is set correctly)
     # - snapshot versions: records the version of the snapshot
-    _tbl_versions: dict[tuple[UUID, int | None], TableVersion]
+    # - anchored versions: records the tbl_id of the anchor table
```

Comment:
Spell out that this is for replicas (it's unclear whether there are other applications).